### PR TITLE
Added to CODEOWNERS and MAINTAINERS files.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
*Issue #, if available:*

*Description of changes:*
Added myself to the repo.

*CheckList:*
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/index-management/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AWSHurneyt** [comment]
@gaiksaya 
You mentioned that you updated the repo permissions in PR https://github.com/opensearch-project/index-management/pull/776. Would you need to do that in this case as well?

**Comment by gaiksaya** [comment]
> @gaiksaya You mentioned that you updated the repo permissions in PR #776. Would you need to do that in this case as well?

Yeah! Will wait for approvers from the maintainers for this PR.

**Review by lezzago — DISMISSED** [review]

**Review by bowenlan-amzn — DISMISSED** [review]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -13,3 +13,7 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Xuesong Luo           | [xluo-aws](https://github.com/xluo-aws)       | Amazon      |
 | Hailong Cui           | [Hailong-am](https://github.com/Hailong-am)   | Amazon      |
 | Binlong Gao           | [gaobinlong](https://github.com/gaobinlong)   | Amazon      |
+| Amardeepsingh Siglani           | [AWSHurneyt](https://github.com/amsiglan)   | Amazon      |
```
Comment (gaiksaya) [inline MAINTAINERS.md:16]:
```suggestion
| Amardeepsingh Siglani           | [amsiglan](https://github.com/amsiglan)   | Amazon      |
```

**Review by bowenlan-amzn — DISMISSED** [review]

**Comment by gaiksaya** [comment]
Permissions are updated now:
image

File: MAINTAINERS.md [hunk 2]
```diff
@@ -13,3 +13,7 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Xuesong Luo           | [xluo-aws](https://github.com/xluo-aws)       | Amazon      |
 | Hailong Cui           | [Hailong-am](https://github.com/Hailong-am)   | Amazon      |
 | Binlong Gao           | [gaobinlong](https://github.com/gaobinlong)   | Amazon      |
+| Amardeepsingh Siglani           | [amsiglan](https://github.com/amsiglan)   | Amazon      |
+| Subhobrata Dey           | [sbcd90](https://github.com/sbcd90)   | Amazon      |
+| Surya Sashank Nistala           | [eirsep](https://github.com/eirsep)   | Amazon      |
+| Thomas Hurney           | [AWSHurneyt](https://github.com/AWSHurneyt)   | Amazon      |
```
Comment (lezzago) [inline MAINTAINERS.md:19]: Can you fix the spacing here to match the above?

Comment (AWSHurneyt) [reply MAINTAINERS.md]: Revised.

**Review by bowenlan-amzn — APPROVED** [review]

**Review by lezzago — APPROVED** [review]

**Pull request merged**